src/constraint/spec.ts

```diff
@@ -538,7 +538,7 @@ export const SPEC_CONSTRAINTS: SpecConstraintModel[] = [
   {
     name: 'scaleZeroMustMatchScaleType',
```

Comment:
Shouldn't this be a part of `scalePropertiesSupportedByScaleType`?  

Plus, I don't know why it is in `spec.ts`.  

I think you should should this.